WordPress/Resources/AppStoreStrings.po

```diff
@@ -34,15 +34,20 @@ msgid ""
 "View the Privacy Notice for California Users at https://automattic.com/privacy/#california-consumer-privacy-act-ccpa.\n"
 msgstr ""
 
-msgctxt "v18.9-whats-new"
+msgctxt "v19.0-whats-new"
 msgid ""
-"New year, new look! We updated comment threads with a shiny new design for author labels, accessory and share buttons, and row and reply highlighting. We also made a few updates to the Block Editor.\n"
+"What time is it? Our brand-new time zone selection screen knows. Get a suggested time zone based on your device and improve search while you’re at it.\n"
 "\n"
-"- When you’re editing Paragraph blocks, you can highlight specific text in any color you’d like.\n"
-"- Brand-new Gallery blocks will default to the new design. You can also auto-convert any old galleries to the new format without a hitch.\n"
-"- The editor now auto-scrolls, so you can always see what you’re typing. No more disappearing text, no more guesswork.\n"
-"- You’ll now see a Comments snippet displayed after your post content in the Reader. This gives you a quick sample of any comments you’ve received and helps you join the conversation faster.\n"
-"- We squashed a bug where the background color wasn’t showing up for pre-formatted blocks on sites that use standard themes.\n"
+"If you’re using a free plan, video uploads are now limited to five minutes per file. Don’t worry, that won’t affect any videos you’ve already uploaded to your site.\n"
+"\n"
+"We’ve also made some changes to the block editor:\n"
+"\n"
+"- We solved translation issues that caused some user interface elements (like text alignment information and the block editor modal window) to only appear in English.\n"
+"- We added language translation for Jetpack and Layout Grid blocks.\n"
+"- We fixed a bug that would reset text formatting to default after hitting backspace. Bold on, friends.\n"
+"- Multi-line block titles are now center-aligned in the inserter menu.\n"
+"\n"
+"Finally, the “Share WordPress with a friend” row is back where it belongs on the Me screen. Reunited and it feels so good.\n"
```

Comment:
```suggestion
"Finally, the \“Share WordPress with a friend\” row is back where it belongs on the Me screen. Reunited and it feels so good.\n"
```

Reply:
Incorrect suggestion after all, since those are curved quotes (U+201C/U+201D), not regular quotes (U+0022), so they don't need escaping!